build.sh

```diff
@@ -187,3 +192,21 @@ if buildAll || hasArg dask_cudf; then
         python setup.py build_ext --inplace
     fi
 fi
+
+# Do not build libcudf_kafka with 'buildAll'
+if hasArg libcudf_kafka; then
+
+    echo "invoking this"
```

Comment:
We don't typically have many echo statements during the build, and I don't mind if they exist. However, instead of using `echo` please add this logger function to the script and use this in its place and use capitalization and ellipses at the end.

```
function logger() {
  echo -e "\n>>>> $@\n"
}
```

Example usage in other build scripts: https://github.com/rapidsai/cudf/blob/branch-0.15/ci/cpu/build.sh#L51

Reply:
I had left those in there by mistake. I will remove them, they shouldn't be there.